### PR TITLE
Allow constructing money from sub-units.

### PR DESCRIPTION
**Pull request by raymondjavaxx**
Hi all,

I have added a pair of methods for working with sub-units that can come in handy for those storing money in sub units.

```python
price = Money.from_sub_units(101, Currency.USD)  # $1.01 USD
total = price * 2
total.sub_units  # 202 (cents)
```
Thank you for crafting this library and reviewing this PR.

-R

### CODE REVIEW

File: money/money.py [hunk 1]
```diff
@@ -29,6 +29,18 @@ def currency(self) -> Currency:
...
+    def from_sub_units(cls, sub_units: int, currency: Currency=Currency.USD):
+        """Creates a Money instance from sub-units."""
+        sub_units_per_unit = CurrencyHelper.sub_unit_for_currency(currency)
+        return cls(Decimal(sub_units) / Decimal(sub_units_per_unit), currency)
+
+    @property
+    def sub_units(self) -> int:
+        """Converts the amount to sub-units"""
+        sub_units_per_unit = CurrencyHelper.sub_unit_for_currency(self.currency)
+        return int(self._round(self.amount, self.currency) * sub_units_per_unit)
```
Comment (nickyr) [inline money/money.py]: why is _round needed here? i think it should have already been rounded in the constructor.

**Review by nickyr — COMMENTED** [review]
Looks really good! Could you please add documentation of these new features in the README?

File: money/money.py (hunk 1 above)
Comment (raymondjavaxx) [reply money/money.py]: Good point. I have updated the code to remove the unnecessary rounding.

**Comment by nickyr** [comment]
Looks great! Pulled it locally to run the tests and pylint and discovered that pylint is complaining about there being too many public methods in the test file now, which I think we should just ignore.

Could you add `too-many-public-methods` to the list here? https://github.com/vimeo/py-money/blob/4b6a06466c96e27e7137818d81947d9718bec83c/money/tests/test_money.py#L10

After that, I'll merge!

(Guess I should set up a travis build on this thing at some point...)

**Comment by raymondjavaxx** [comment]
Cool. I just added `too-many-public-methods` to the list.

**Review by nickyr — APPROVED** [review]
Thanks!

**Comment by nickyr** [comment]
Release is up on PyPi as 0.3.0. LMK if it's not working for you, I had so many PyPi issues (as usual)

**Pull request merged**